app/Exceptions/Handler.php

```diff
@@ -64,12 +65,21 @@ public function render($request, Throwable $exception)
             return $this->getResponse(Response::HTTP_UNPROCESSABLE_ENTITY);
         }
 
-        $this->logger->setTraceId($request->headers->get(HeadersMiddleware::X_TRACE_ID));
-        $this->logger->error(
-            Response::$statusTexts[Response::HTTP_INTERNAL_SERVER_ERROR],
-            ['exception' => $exception]
+        // $this->logger->setTraceId($request->headers->get(HeadersMiddleware::X_TRACE_ID));
+        // $this->logger->error(
+        //     Response::$statusTexts[Response::HTTP_INTERNAL_SERVER_ERROR],
+        //     ['exception' => $exception]
+        // );
+        /** @var Log $logger */
+
+        $logger = new LogAdapter();
```

Comment:
Instead of instatiate manually, wouldn't be better to use the app to create the instance?

```php
$logger = app(Log::class);
```

Reply:
ok